crates/bevy_text/src/lib.rs

```diff
@@ -38,6 +38,25 @@ pub type DefaultTextPipeline = TextPipeline<Entity>;
 #[derive(Default)]
 pub struct TextPlugin;
 
+/// `TextPlugin` settings
+#[derive(Resource)]
+pub struct TextSettings {
```

Comment:
This kinda feels like it should be named `FontAtlasSettings`, but I guess we could use it for any other text related setting in the future

Reply:
My initial idea was to use it for any related text settings in the future. Especially since now it is passed through the whole text pipeline it would be an easy place to add new settings.